src/ec/suite_b/public_key.rs

```diff
@@ -28,10 +28,10 @@ use untrusted;
 ///
 /// [NIST SP 800-56A, revision 2]:
 ///     http://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-56Ar2.pdf
-pub fn parse_uncompressed_point<'a>(ops: &PublicKeyOps,
-                                    input: untrusted::Input<'a>)
-                                    -> Result<(Elem, Elem),
-                                              error::Unspecified> {
+pub fn parse_uncompressed_point(ops: &PublicKeyOps,
+                                input: untrusted::Input)
+                                -> Result<(Elem, Elem),
+                                          error::Unspecified> {
```

Comment:
This can fit inside 80 columns now. (That is the policy, right? I didn't find it in STYLE.md.)